common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -252,6 +266,8 @@ main() {
 VERBOSE=false
 PLATFORM_INIT=true
 DELETE_LOGS=false
+CONTROL_IFACE=enp0s25
+DATA_IFACE=enx00ee22aa28ef
```

Comment:
I would default them to empty, and only set them if requested by the parameter.

Actually, does it make a lot of sense to include control-iface in prplmesh-utils.sh? I would actually set it from network-manager, or not even use a static IP but have a DHCP server in my setup.

Reply:
I prefer to keep them default to the testbed we are using now to make life easy for our QA.
We could do that in the network manager, if you mean the equivalent of /etc/network/interfaces, but then we need to do the same for each laptop we use for certification and it is one more step someone might get wrong. Having the default in the script (which is currently used only for certification or dummy mode which still works since these interfaces are not there) makes sense to me, at least at this point in the project.
If you still disagree, I'll consider changing it :)